topics/go/algorithms/data/hash/hash.go

```diff
@@ -0,0 +1,107 @@
+// Package hash implements a hash table
+package hash
+
+import (
+	"fmt"
+	"hash/maphash"
+)
+
+const (
+	numBuckets = 256 // Number of buckets in hash
+)
+
+// entry in the hash table
```

Comment:
All comments need to be proper sentence structure and complete sentences. Since this is teaching material, we will comment more than not. Make sure the comments are being very clear.

// Entry represents a value we will store in the hash table.

Reply:
OK. IMO the upper case `Entry` might confuse.